src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/SearchByGEController.java

```diff
@@ -54,5 +59,47 @@ public String search(@RequestParam(name = "college", required = true) String col
         return "search/byge/results";
     }
     
+    
+    @GetMapping("/search/byge/multiquarter")
+    public String instructor(Model model, SearchByGEMultiQuarter SearchByGEMultiQuarter) {
+        model.addAttribute("searchByGEMultiQuarter", new SearchByGEMultiQuarter());
+        model.addAttribute("quarters", Quarter.quarterList("W20", "F83"));
+        return "search/byge/multiquarter/search";
+    }
+
+
+    @GetMapping("search/byge/multiquarter/results")
+    public String SearchByGEMultiQuarter(@RequestParam(name = "college", required = true) String college,
+    @RequestParam(name = "area", required = true) String area,
+    @RequestParam(name = "beginQ", required = true) int beginQ,
+    @RequestParam(name = "endQ", required = true) int endQ, 
+    Model model,
+    SearchByGEMultiQuarter SearchByGEMultiQuarter) {
```

Comment:
I don't believe this is necessary (be sure to test it though!)

```suggestion
    Model model) {
```